ocaml/lambda/translcore.ml

```diff
@@ -280,18 +301,28 @@ let event_before ~scopes exp lam =
 let event_after ~scopes exp lam =
   Translprim.event_after (of_location ~scopes exp.exp_loc) exp lam
 
-let event_function ~scopes exp lam =
+let event_function_impl ~scopes loc env lam =
```

Comment:
`_impl` is an uninformative name. How about naming this one `event_function` and then naming the thing that's currently named `event_function` to `event_function_expr` to more strongly suggest a correspondence to `event_function_rhs`?